--- a/cmake/FindIconv.cmake
+++ b/cmake/FindIconv.cmake
@@ -23,11 +23,6 @@
 IF(CMAKE_SYSTEM_NAME MATCHES "SunOS")
   # There is some libiconv.so in  /usr/local that must
   # be avoided, iconv routines are in libc  
-ELSEIF(APPLE)
-  find_library(ICONV_LIBRARIES NAMES iconv libiconv PATHS
-               /usr/lib/
-               NO_CMAKE_SYSTEM_PATH)
-    SET(ICONV_EXTERNAL TRUE)
 ELSE()
   find_library(ICONV_LIBRARIES NAMES iconv libiconv libiconv-2)
   IF(ICONV_LIBRARIES)